uk.co.compucorp.civicrm.hrleaveandabsences/tests/phpunit/CRM/HRLeaveAndAbsences/BAO/AbsenceTypeTest.php

```diff
@@ -22,6 +22,13 @@ public function setUpHeadless() {
     return \Civi\Test::headless()->installMe(__DIR__)->apply();
   }
 
+  public function setUp() {
+    // We delete everything two avoid problems with the default absence types
+    // created during the extension installation
+    $tableName = CRM_HRLeaveAndAbsences_BAO_AbsenceType::getTableName();
```

Comment:
This is a bad practice for tests and can make the test to fail if we decide to remove the default types in the future. In doing unit tests, we have to make sure to isolate each test in itself. That is, a test cannot depend on something other than what it or the setup method creates.